test/fixtures/mjs-support/src/App.js

```diff
@@ -1,62 +1,35 @@
 import React, { Component } from 'react';
-import ApolloClient, { gql } from 'apollo-boost';
```

Comment:
If I remember correctly the purpose of this test was to ensure `mjs` files were handled correctly. Apollo was shipping `mjs` files (at least back then, not sure about now) which is why we used it for the test. I don't know why we were actually making a query to a remote GraphQL API though. I think we really just need to make sure we can properly import a package that ships `mjs` files.

Reply:
Yeah I think you're correct. I removed all the other packages so we could just use graphql by itself. The `graphql` package has .mjs (unless i mis read, was late at night). I'll double check that's the case.

Reply:
Seems legit?

![image](https://user-images.githubusercontent.com/6355370/58386720-96124300-7fb8-11e9-8dc8-2dccaca83914.png)